packages/neovim/src/utils/findNvim.ts

```diff
@@ -113,19 +117,66 @@ function compareVersions(a: string, b: string): number {
   return 0;
 }
 
+function getPlatformPaths() {
+  const paths = new Set<string>();
+  const { PATH, USERPROFILE, LOCALAPPDATA, PROGRAMFILES, HOME } = process.env;
+
+  const normalizePath = (path: string) => normalize(path.toLowerCase());
+
+  PATH?.split(delimiter).forEach(p => paths.add(normalizePath(p)));
+
+  // Add common Neovim installation paths not always in the system's PATH.
+  if (windows) {
+    // Scoop common install location
+    if (USERPROFILE) {
+      paths.add(normalizePath(`${USERPROFILE}\\scoop\\shims`));
+    }
+    paths.add(normalizePath('C:\\ProgramData\\scoop\\shims'));
```

Comment:
no need to use \ slashes, even on windows
```suggestion
    paths.add(normalizePath('C:/ProgramData\/scoop/shims'));
```